sonic_platform_base/component_base.py

```diff
@@ -108,6 +108,9 @@ def update_firmware(self, image_path):
         Args:
             image_path: A string, path to firmware image
 
+        Returns:
+            False if image not found.
+
```

Comment:
can we have return 'False' for any failure. see the definition of install_firmware()?

```
  Returns:
            A boolean, True if install was successful, False if not
```